test/UploadedFileTest.php

```diff
@@ -274,15 +275,17 @@ public function testGetStreamRaisesExceptionWhenErrorStatusPresent($status)
      */
     public function testMoveToCreatesStreamIfOnlyAFilenameWasProvided()
     {
+        $orgFile = tempnam(sys_get_temp_dir(), 'ORG');
         $this->tmpFile = tempnam(sys_get_temp_dir(), 'DIA');
 
-        $uploadedFile = new UploadedFile(__FILE__, 100, UPLOAD_ERR_OK, basename(__FILE__), 'text/plain');
+        $original = file_get_contents($orgFile);
+
+        $uploadedFile = new UploadedFile($orgFile, 100, UPLOAD_ERR_OK, basename($orgFile), 'text/plain');
         $uploadedFile->moveTo($this->tmpFile);
 
-        $original = file_get_contents(__FILE__);
-        $test     = file_get_contents($this->tmpFile);
+        $contents = file_get_contents($this->tmpFile);
 
-        $this->assertSame($original, $test);
+        $this->assertSame($original, $contents);
     }
```

Comment:
This test should be left as it was, to avoid hiding accidental BC breaks.

Reply:
This test specifies __FILE__ as resource.
It means UploadedFile::moveTo removes __FILE__ after moving this TestCaseClass file.

Reply:
Hmm, then we should not operate with `__FILE__`, since that is the test itself :thinking:

Reply:
Can we put some contents in `$orgFile`?

Reply:
A `"Hello"` written to it would be more than enough :-)